mantidproject/mantid
# Use full wavelength range in HRPD efficiency correction

**Description of work.**

Fixes the issue with the 100-200ms window on HRPD focussing to all zeroes. Also gives control of the integration to the scientists in the script.

**To test:**

Run the example in the issue and see that there is a sensible spectrum of Bragg peaks and not zeros.

Fixes #27238.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
